### PR TITLE
Modified runserver to be able to use in production

### PR DESCRIPTION
**Pull request by lsetiawan**
I modified the odm2 timeseries runserver, so that I can use for both development and production. @ocefpaf Let me know if I'm breaking Pep8 and if I do, how can I do better? Basically I need to get the app variable for wsgi to use, and the `argparse` stuff is for development user input. Thanks.

### CODE REVIEW

File: wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py [hunk 1]
```diff
@@ -15,36 +14,34 @@
...
         connection = config['Database']['Connection_String']
+        return connection
+
+parser = argparse.ArgumentParser(description='start WOF for an ODM2 database.')
+parser.add_argument('--config',
+                   help='Configuration file', default='odm2_config_timeseries.cfg')
+parser.add_argument('--port',
+                   help='Open port for server."', default=8080, type=int)
+args = parser.parse_args()
+print(args)
```
Comment (ocefpaf) [inline wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py]: Do we still need the `print` debug?

File: wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py [hunk 2]
```diff
@@ -15,36 +14,34 @@
...
 
-def startServer(conf='odm2_config_timeseries.cfg', openPort=8080):
-
+def get_connection(conf):
     # Parse connection from config file
     config = configparser.ConfigParser()
     with open(conf, 'r') as configfile:
         config.read_file(configfile)
         connection = config['Database']['Connection_String']
+        return connection
```
Comment (ocefpaf) [inline wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py]: Just to be on the safe side I would  place the `return` out of the `with` block.

**Comment by ocefpaf** [comment]
> Let me know if I'm breaking Pep8 and if I do, how can I do better?

Travis-CI is reporting that in the `allow_failure` section. See https://travis-ci.org/ODM2/WOFpy/jobs/231591322

But we have so many violations now that it is pointless to fix them all. BTW your code looks fine. (You can always run `flake8` on the script to check.)

File: wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py (hunk 1 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py]: Not really. I will remove this

**Comment by lsetiawan** [comment]
@ocefpaf Great! Thanks so much 😸

**Comment by emiliom** [comment]
> I modified the odm2 timeseries runserver, so that I can use for both development and production. Basically I need to get the app variable for wsgi to use, and the argparse stuff is for development user input. 

I looked at the code changes. Sorry, I didn't go back to your wofpy instructions (and my memory is too hazy) to remember exactly how runserver is used differently in the two contexts, and why these changes are needed (it worked before ...) or what improvement they provide. I assume these are *improvements*, not bug fixes? Can you describe briefly what they improve over the previous code?

**Comment by lsetiawan** [comment]
@emiliom This is an improvement. Previous code doesn't allow me to import `app` to be used in `wsgi.py` so that `uwsgi` handle the whole server deployment thing. So rather than creating a whole other `runserver.py` the person deploying the app can just use `runserver_odm2_timeseries.py ` straight out of the box for production. They only need to configure the `.cfg` that correspond to the dao. Thanks.

**Comment by emiliom** [comment]
Thanks. I thought that was already possible, but again, my hazy memory :(

I was about to merge, but I see that @ocefpaf already did.

**Comment by ocefpaf** [comment]
> I was about to merge, but I see that @ocefpaf already did.

Oops. Sorry. The changes made sense and I got trigger happy with the merge button.

**Pull request merged**